_config.php

```diff
@@ -17,14 +17,10 @@
  * - CMS_PATH: Absolute filepath, e.g. "/var/www/my-webroot/cms"
  */
 call_user_func(function () {
-    define('CMS_PATH', __DIR__);
```

Comment:
Hmmm, by no longer defining these we need to mention the change in the upgrading guide - it's no longer an opt-in feature. I'll write up something